test/unit/src/common/components/repository-list/t_repository-list.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow } from 'enzyme';
+
+import { RepositoriesList } from '../../../../../../src/common/components/repositories-list';
+import { Table, Body } from '../../../../../../src/common/components/vanilla/table-interactive';
+
+// state fixtures
+const initialState = {
+  hasNoRegisteredNames: true,
+  hasNoConfiguredSnaps: true,
+  snaps: {
+    isFetching: false,
+    success: false,
+    error: {},
+    snaps: [{
+      git_repository_url: 'https://github.com/earnubs/bsi-test-i'
+    }, {
+      git_repository_url: 'https://github.com/earnubs/bsi-test-ii'
+    }, {
+      git_repository_url: 'https://github.com/earnubs/bsi-test-iii'
+    }, {
+      git_repository_url: 'https://github.com/earnubs/bsi-test-iv'
+    }, {
+      git_repository_url: 'https://github.com/earnubs/bsi-test-v'
+    }]
+  },
+  registerName: {},
```

Comment:
This makes me wonder if we used defaultProps more could most of these fixtures go away?